docs/configuration.md

```diff
@@ -29,29 +27,11 @@ spec:
   templateValidator:
     replicas: 2
   featureGates:
-    deployTektonTaskResources: true
     deployVmConsoleProxy: true
-  tektonPipelines:
-    namespace: kubevirt
-  tektonTasks:
-    namespace: kubevirt
 ```
 
 ## Annotations
 
-### VM Console Proxy
-
-This annotation is used by VM console proxy operand.
-
-```
-apiVersion: ssp.kubevirt.io/v1beta1
-kind: SSP
-metadata:
-  name: ssp-sample
-  namespace: kubevirt
-spec: {}
-```
-
```

Comment:
Is this intentional? Shouldn't be in a separate PR in any case?

Reply:
Yes, this part of documentation is no longer valid.
I will change the commit message and PR's description. IMHO this is a simple change and a separate PR is not needed.

Reply:
That will work for me. Thanks!

Reply:
done